content/gsoc-2025-selected-projects.md

```diff
@@ -0,0 +1,42 @@
++++
+path = "2025/05/08/gsoc-2025-selected-projects"
+title = "Announcing Google Summer of Code 2025 selected projects"
+authors = ["Jakub Beránek"]
+
+[extra]
+team = "the mentorship team"
+team_url = "https://www.rust-lang.org/governance/teams/launching-pad#team-mentorship"
++++
+
+The Rust Project is [participating][gsoc blog post] in [Google Summer of Code (GSoC) 2025][gsoc] again this year. GSoC is a global program organized by Google that is designed to bring new contributors to the world of open-source.
```

Comment:
"in Google Summer of Code (GSoC) 2025 again" is kind of weird - we're participating in GSoC again, not GSoC 2025 again.

Reply:
Good point, I will reword this in the final version.